blog/migrations/0012_auto_20200625_1631.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.11 on 2020-06-25 16:31
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailforms', '0004_add_verbose_name_plural'),
+        ('wagtailredirects', '0006_redirect_increase_max_length'),
+        ('wagtailcore', '0045_assign_unlock_grouppagepermission'),
+        ('blog', '0011_auto_20191204_1137'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='blogarticlepage',
+            name='categories',
+        ),
+        migrations.DeleteModel(
+            name='BlogCategoryPage',
+        ),
+        migrations.DeleteModel(
+            name='BlogCategorySnippet',
+        ),
+    ]
```

Comment:
I'm just curious if you checked if everything is working with migration when you deleting categories from existing articles?

Reply:
Holy Jesus jumping jacks in jelly jeans, I completely forgot about checking that!
But I'm glad to inform you everything works in order. 👍